source/js/searchRecipe.js

```diff
@@ -0,0 +1,48 @@
+function searchRecipe(key,apiKey){
+  let query = "https://api.spoonacular.com/recipes/complexSearch?apiKey="+apiKey+"&query="+key;
```

Comment:
Codacy has a fix for the issue: ['query' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-fa21-group7/cse110-fa21-group7/pullRequest?prid=8446960)
```suggestion
  const query = "https://api.spoonacular.com/recipes/complexSearch?apiKey="+apiKey+"&query="+key;
```